ui/src/components/Organization/List.tsx

```diff
@@ -9,17 +9,33 @@ interface Props {
   organizations: Organization[];
 }
 
+const defaultBand = (brand: string, name: string) => {
+  if (brand && brand.length) {
+    return <img className={c('brand')} src={brand} />;
+  } else {
+    const firstLetter = name.split('')[0];
+    return (
+      <div className={c('default-brand')}>
+        <span className={c('default-icon')}>{firstLetter}</span>
+      </div>
+    );
+  }
+};
+
 const OrgItem: React.FC<Organization> = (p) => (
-  <div className={c('content')} key={p.id}>
-    <div className={c('section1')}>
-      <img className={c('logo')} src={p.square_logo} />
-      <span className={c('name')}>{p.name}</span>
+  <div className={c('container')}>
+    <div className={c('content')} key={p.id}>
```

Comment:
Please take a look if we need a `content` immediately following, and as only child of `container`. These are mostly not needed.